cmd/zpool/zpool.d/ses

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Print SCSI Enclosure Services (SES) info. The output is dependent on the name
+# of the script/symlink used to call it.
+#
+helpstr="
+enc:		Show disk enclosure w:x:y:z value.
+slot:		Show disk slot number as reported by the enclosure.
+encdev:		Show the /dev/sg* device for the enclosure associated with the disk slot.
+fault_led:	Show the value of the disk enclosure slot fault LED.
+locate_led:	Show the value of the disk enclosure slot locate LED.
+ses:		Show disk's enclosure, enclosure dev, slot number, and fault/locate LED values."
+
+script=$(basename "$0")
+if [ "$1" = "-h" ] ; then
```

Comment:
Actually, nevermind. I don't see any other things that you could pull into a lib style shell script. Just ignore this.